main.js

```diff
@@ -0,0 +1,83 @@
+//bai1
+const getNumber = n => {
+    n == Number
+    for (i = 1; i <= 10; i++) {
+        console.log('${n} * ${i} =', i)
+    }
+}
+
+//bai2
+const takeEvenNumber = n => {
+    if (n >= 1 && n <= 30) {
+        for(let i =2; i <= n; i += 2)
+        console.log(i)
+    }
+}
+
+//bai3
```

Comment:
```suggestion
// Bài 3
```
Và hãy dùng tiếng anh thay vì tiếng Việt